README.md

```diff
@@ -219,6 +219,12 @@ easy_install gspread
     nosetests
     ```
 
+## Testing environment setup
+
+    pip install nose
+    pip install requests['security']
+    pip install oath2client
+
```

Comment:
The common way of doing this is by having a `requirements.txt` file in the `tests` subdirectory.